tests/e2e/steps_permissionless_ics.go

```diff
@@ -52,7 +53,11 @@ func stepsPermissionlessICS() []Step {
 						TopN: 0,
 					},
 				},
-				State: State{},
+				State: State{
+					ChainID("provi"): e2e.ChainState{
+						ConsumerChains: &map[ChainID]bool{"cons2": true}, // Consumer chain "consu1" is now removed
```

Comment:
Why does the comment state that it "is now removed"? It seems that it was just added.

Reply:
wrong line, it was not even started yet.